src/components/ButtonPanel.js

```diff
@@ -1,42 +1,47 @@
+import PropTypes from 'prop-types';
 /* eslint-disable no-unused-vars */
 import Button from './Button';
 
-const signCalc = (sign) => < Button name = {sign} />;
-
-const ButtonPanel = () => {
+const mapBtnsToPanel = (arr, clickHandler) => {
+  const res = arr.map(
+    curr => {
+      let color;
+      const signs = ['AC', '÷', 'x', '-', '=', '+'];
+      if (signs.includes(curr)) {
+        color = 'orange';
+      } else {
+        color = 'grey';
+      }
+      return (<Button color={color} name={curr} clickHandler={() => clickHandler(curr)} />);
+    },
+  );
+  return res;
+};
+const displayAllBtns = (rowsArr) => rowsArr.map((row, index) => (
+  <div className={`row${index}`}>{ row }</div>
+));
+const ButtonPanel = ({ clickHandler }) => {
   const group1 = ['AC', '+/-', '%', '÷'];
-  const group2 = ['7', '8', '9', 'X'];
+  const group2 = ['7', '8', '9', 'x'];
   const group3 = ['4', '5', '6', '-'];
   const group4 = ['1', '2', '3', '+'];
   const group5 = ['0', '.', '='];
-  const sign1 = group1.map(signCalc);
-  const sign2 = group2.map(signCalc);
-  const sign3 = group3.map(signCalc);
-  const sign4 = group4.map(signCalc);
-  const sign5 = group5.map(signCalc);
+  const signArr = [
+    mapBtnsToPanel(group1, clickHandler),
+    mapBtnsToPanel(group2, clickHandler),
+    mapBtnsToPanel(group3, clickHandler),
+    mapBtnsToPanel(group4, clickHandler),
+    mapBtnsToPanel(group5, clickHandler),
+  ];
   return (
     <div className = "calc">
-       <div className = "btn-show">
-          {sign1}
-       </div>
-
-       <div className = "btn-show">
-          {sign2}
-       </div>
-
-       <div className = "btn-show">
-          {sign3}
-       </div>
-
-       <div className = "btn-show">
-          {sign4}
-       </div>
-
-       <div className = "btn-show">
-          {sign5}
-       </div>
+       {displayAllBtns(signArr)}
       </div>
   );
 };
-/* eslint-enable no-unused-vars */
```

Comment:
## [OPTIONAL] Stretch Requirement

- [ ] By disabling ESLint rules,  don't you think it rids you the opportunity to write much cleaner code and suppresses the potential warnings you’d be getting, thus missing out on the education that ESLint rules provide? 😟  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no-ignore-linters.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/)).